model/src/id/mod.rs

```diff
@@ -168,6 +168,24 @@ impl<T> Id<T> {
         self.value.get()
     }
 
+    /// Return the inner value directly as a [`NonZeroU64`].
+    ///
+    /// # Examples
+    ///
+    /// Crate an ID with a value and then confirmt its inner value:
+    ///
+    /// ```
+    /// use std::num::NonZeroU64;
+    /// use twilight_model::id::{marker::ChannelMarker, Id};
+    ///
+    /// let channel_id = Id::<ChannelMarker>::new(7);
+    ///
+    /// assert_eq!(NonZeroU64::new(7).unwrap(), channel_id.get_nonzero());
+    /// ```
+    pub const fn get_nonzero(self) -> NonZeroU64 {
```

Comment:
I think calling it `inner` would be better. 

```suggestion
    pub const fn inner(self) -> NonZeroU64 {
```

Reply:
Thanks, totally agreed this is a better naming 👍